tool-openssl/x509_test.cc

```diff
@@ -21,17 +24,18 @@
 
 size_t createTempFILEpath(char buffer[PATH_MAX]);
 
+void RemoveFile(const char* path);
+
 X509* CreateAndSignX509Certificate() {
   bssl::UniquePtr<X509> x509(X509_new());
   if (!x509) return nullptr;
 
-  // Set validity period for 1 year
+  // Set validity period for 30 days
```

Comment:
Not an issue, but was curious why the change to 30 days?

Reply:
30 days is the default for OpenSSL